concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
```

Comment:
It should look like:
```
ProtocolConstants = namedtuple(
    "ProtocolConstants",
    [
        "concent_messaging_time",
        "subtask_verification_time",
        "force_acceptance_time",
        "token_expiration_time",
    ]
)

Reply:
Actually it is not used anywhere else beyond `print` function. Why don't you want to use dict from response directly ?

Reply:
It's nice to have a namedtuple if your dict has strictly defined fields. It's similar in purpose to an enum. You could use strings instead of enum values but it you mistype it or use one that does not exist, Python can notice and report an error.